emulator/blockchain.go

```diff
@@ -888,9 +889,13 @@ func (b *Blockchain) GetTransactionResult(txID flowgo.Identifier) (*access.Trans
 		return nil, err
 	}
 
+	statusCode := codes.OK
+	if storedResult.ErrorCode > 0 {
+		statusCode = codes.Canceled
+	}
```

Comment:
The docs that say it's a grpc status code are misleading. I assume that's something to do with the fact that it's accessed over the grpc api (🤷). It's actually one of these enum values
https://github.com/onflow/flow-go/blob/37ece4cd39961442ebae8d622b63fb5a8cf8da18/model/flow/transaction.go#L406-L417

so it should either be `flow.TransactionStatusSealed` or `flow.TransactionStatusExecuted`.